CI failure looks related

```
--- a/Documentation/helm-values.rst
+++ b/Documentation/helm-values.rst
@@ -2652,6 +2652,14 @@
      - Controls server listener for ipv6
      - object
      - ``{"enabled":true}``
+   * - :spelling:ignore:`hubble.ui.httpRoute`
+     - hubble-ui GatewayAPI HTTPRoute configuration.
+     - object
+     - ``{"annotations":{},"enabled":false,"hostnames":["chart-example.local"],"labels":{},"parentRefs":[{"name":"cilium-gateway","namespace":"kube-system"}]}``
+   * - :spelling:ignore:`hubble.ui.httpRoute.parentRefs`
+     - parentRefs define which Gateways this HTTPRoute attaches to.
+     - list
+     - ``[{"name":"cilium-gateway","namespace":"kube-system"}]``
    * - :spelling:ignore:`hubble.ui.ingress`
      - hubble-ui ingress configuration.
      - object
make: *** [Makefile:133: check] Error 1
HINT: to fix this, run 'make -C Documentation update-helm-values'
```

--- a/Documentation/helm-values.rst
+++ b/Documentation/helm-values.rst
@@ -2652,6 +2652,14 @@
      - Controls server listener for ipv6
      - object
      - ``{"enabled":true}``
+   * - :spelling:ignore:`hubble.ui.httpRoute`
+     - hubble-ui GatewayAPI HTTPRoute configuration.
+     - object
+     - ``{"annotations":{},"enabled":false,"hostnames":["chart-example.local"],"labels":{},"parentRefs":[{"name":"cilium-gateway","namespace":"kube-system"}]}``
+   * - :spelling:ignore:`hubble.ui.httpRoute.parentRefs`
+     - parentRefs define which Gateways this HTTPRoute attaches to.
+     - list
+     - ``[{"name":"cilium-gateway","namespace":"kube-system"}]``
    * - :spelling:ignore:`hubble.ui.ingress`
      - hubble-ui ingress configuration.
      - object

